adr/2021_04_15_using_iri_instead_of_code_id.md

```diff
@@ -0,0 +1,46 @@
+# Using iri instead of code/id in requests
+
+* Status: proposed
+* Date: 2021-04-15
+
+## Context and Problem Statement
+
+ApiPlatform provides and recommends using IRI as identifier. That identifier is more clearly than classic id and 
+contain more information. On resources, ApiPlatform handles IRI out of the box.
+During creation our new API in many places we decided to use commands while creating our endpoints. For certain cases
+this solution is more flexible than classic approach, but it doesn't handle transformation IRI to proper `code`/`id`.
+In the past we decided to use `code`/`id` instead of IRI. Now we try to unify API and replace this approach for solution with using IRI.
+Main problem is sent request with field as IRI and creation command without IRI, but with `id`/`code`. 
+It's needed for proper handling behaviour in command handler.
+
+## Considered Options
```

Comment:
I would be for adding the second option to use ids and codes to this consideration section and describing its pros and cons